docs/content/posts/2023-08-03-connector-updates.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "DynamoDB, MongoDB and OpenSearch externalized connectors"
```

Comment:
```suggestion
title: "Announcing three new Apache Flink connectors, the new connector versioning strategy and externalization"
```

Reply:
I am not 100% on this, what do you think?

Reply:
It sounds better